cmd/kusk/cmd/install.go

```diff
@@ -37,15 +37,16 @@ import (
 )
 
 var (
-	noApi                         bool
-	noDashboard                   bool
-	noEnvoyFleet                  bool
-	releaseName, releaseNamespace string
+	noApi            bool
+	noDashboard      bool
+	noEnvoyFleet     bool
+	releaseNamespace string
 )
 
+const releaseName = "kusk-gateway"
+
 func init() {
 	rootCmd.AddCommand(installCmd)
-	installCmd.Flags().StringVar(&releaseName, "name", "kusk-gateway", "installation name")
```

Comment:
This flag needs to remain to allow users to override the release name.

Reply:
I removed it becuase there is a bug introduced by it: 

https://github.com/kubeshop/kusk-gateway/blob/main/cmd/kusk/cmd/install.go#L108

We are using the flag to also search for the chart, so if a user uses a custom name, the command stops working 

```sh
kusk install --name custom
⨯ Installing kusk gateway (error: process error: exit status 1
output: Release "custom" does not exist.
``` 

Should I still add it? @kylehodgetts

Reply:
Also, I think `upgarde` command should not receive a releaseName, right? It should just upgrade the existing kusk-gateway distribution, I'd think. So we would have to add logic to look for the existing kusk-gateway in the cluster, get it namespace 🤔

Reply:
`releaseName` here is used to pull images. If we use it dynamically the users will end up with an error